services/sentry/lib/eventReducer.js

```diff
@@ -24,6 +31,18 @@ function reduce(channel, initialAggr, ev) {
 		}
 	}
 
+	if (ev.type === 'PAY') {
+		const { outputs } = ev
+		const publishers = Object.keys(outputs)
+		publishers.forEach(publisher => {
+			aggr.events.IMPRESSION = mergeImpressionEv(
```

Comment:
i'm not sure about this, maybe we should introduce `aggr.events.PAY` to keep this separate
it should work automagically cause of https://github.com/AdExNetwork/adex-validator/blob/master/services/validatorWorker/lib/mergeAggrs.js#L53

Reply:
okay then